ofxsMaskMix.h

```diff
@@ -192,16 +192,16 @@ ofxsUnPremult(const PIX *srcPix,
         unpPix[0] = 0.f;
         unpPix[1] = 0.f;
         unpPix[2] = 0.f;
-        unpPix[3] = srcPix[0] / (float)maxValue;
```

Comment:
I don't see the point in casting the left operand of "/". Type promotion is part of the C language.